src/app/zap-templates/zcl/data-model/chip/network-commissioning-cluster.xml

```diff
@@ -155,7 +159,10 @@ limitations under the License.
             <description>Add or update the credentials for a given Wi-Fi network.</description>
             <arg name="SSID" type="octet_string" length="32"/>
             <arg name="Credentials" type="octet_string" length="64"/>
```

Comment:
I _think_ the spec makes `Credentials` an optional field, in that if PDC is supported and NetworkIdentity is provided then Credentials can be omitted, right?

Unfortunately, that's an API-breaking change for the SDK, as far as I can tell.  We need to figure out how we're going to handle that....

Reply:
Yes it's meant to be an either or, though I suppose we could do a workaround like ignore the value of `Credentials` (or require it to be the empty string) when `NetworkIdentity` is provided. We would have to update the spec to reflect that.

Reply:
We need to make sure that this is covered by spec, for sure. I think we are fine for now, since the Credentials field is already mandatory, but can be ZERO.

![image](https://github.com/project-chip/connectedhomeip/assets/1313847/555a390a-490b-46ab-b166-bd6b836e7a9d)

Reply:
Filed https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/7938 to track the spec change for this.